src/features/klant/klant-details/fetch-klant.ts

```diff
@@ -0,0 +1,82 @@
+import {
+  fetchKlantByIdOk2,
+  findKlantByIdentifierOpenKlant2,
+} from "@/services/openklant2";
+import {
+  fetchKlantByIdOk1,
+  fetchKlantByIdentifierOpenKlant1,
+} from "@/services/openklant1";
+import {
+  registryVersions,
+  useSystemen,
+  type Systeem,
+} from "@/services/environment/fetch-systemen";
+import type { Klant } from "@/services/openklant/types";
+
+const { systemen, defaultSysteem } = useSystemen();
```

Comment:
Dit is niet handig, want er is geen garantie dat

- de systemen al opgehaald zijn als ensureKlantForBsn aangeroepen wordt
- de systemen niet tussentijds gewijzgd zijn als ensureKlantForBsn op een later moment aangeroepen wordt.

Op andere plekken geven we de lijst met systemen mee als parameter. Kan je dat als voorbeeld gebruiken?